docs/accessibility/screen-readers.md

```diff
@@ -0,0 +1,247 @@
+
+### VoiceOver (macOS)
+
+- VoiceOver is included with MacOS.
+- Commonly paired with: Safari
+- [Keyboard shortcuts (from Deque)](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts)
+
+#### Setting up your Mac and Safari
+
+Before testing with VoiceOver, you may need to enable keyboard navigation—for both your Mac and Safari.
+
+To turn on keyboard navigation for your Mac:
+
+1. Go to the Keyboard section in System Settings.
+2. Turn on the "Keyboard navigation" switch within this section.
+
+(For MacOS versions prior to MacOS 13 Ventura, enter the Shortcuts tab panel in the System Settings' Keyboard section and check the "Use keyboard navigation..." checkbox.)
+
+In Safari, you may need the additional step of enabling the browser's tab navigation:
+
+1. Open Safari’s Preferences/Settings window
+2. Enter the Advanced tab.
+3. Check the "Press Tab to highlight each item on a webpage" checkbox.
+
+#### Getting started with VoiceOver
+
+To toggle VoiceOver on and off, press <kbd>Command-F5</kbd>. Or, if you have a Touch ID keyboard, hold down <kbd>Command</kbd> and quickly press the <kbd>Touch ID</kbd> key three times.
+
+To customize VoiceOver settings (e.g., the voice), open the VoiceOver Utility on your computer (e.g., via <kbd>Command-Space</kbd> and then typing "voiceover utility").
+
+By default, the <kbd>Ctrl-Opt</kbd> keypress combo is what's called the <kbd>VO</kbd> modifier. You can execute many VoiceOver commands with this modifier. For example, <kbd>VO-H</kbd> (or <kbd>Ctrl-Opt-H</kbd>) launches VoiceOver Help (which can then be exited via <kbd>Esc</kbd>).
+
+#### Using the transcript window and muting VoiceOver
+
+To visually reinforce what you’re hearing, VoiceOver’s transcript window reflects the content currently being read. If VoiceOver's audio is too distracting (e.g., maybe you're presenting it in a meeting and want to hear the other participants), you can even mute it via the "Mute speech" checkbox in the VoiceOver Utility’s Speech tab. The transcript window will continue to appear and display content as if it were being read aloud.
+
+While speech is muted, the shortcut for continually reading page content (<kbd>VO-A</kbd>) will be deactivated, though you can still use the <kbd>↑ → ↓ ←</kbd> keys to move through and test non-interactive content.
+
+While transcript windows can be helpful, there are cases where the audio may not match a transcript exactly. So, when testing, confirm any issues with the audio on.
+
+### NVDA (Windows)
+
+- [Download and install](https://www.nvaccess.org/download/) NVDA to use it. It’s free.
+- Commonly paired with: Firefox (and Chrome, if you don’t have access to JAWS)
+- [Keyboard shortcuts](https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts) (from Deque)
+
+#### Getting started
+
+You can start NVDA via any of the following methods:
+
+- Open it in the Windows Start menu.
+- Press the <kbd>Windows</kbd> key, type <code>nvda</code> and then <kbd>Enter</kbd>.
```

Comment:
Should `nvda` be capitalized here?

Reply:
It works either way, so I didn't capitalize it on purpose. But, honestly, I might as well!

Reply:
I'll do the same for "jaws" below.